sensors/altimeter/altimeter.ino

```diff
@@ -0,0 +1,21 @@
+#include "altimeter.h"
+#include <SPI.h>
+// https://github.com/sparkfun/MPL3115A2_Breakout/tree/V_H1.1_L1.2.0/Libraries/Arduino
+#include <SparkFunMPL3115A2.h>
+
+MPL3115A2 altimeter;
+
+Altimeter::Altimeter(const int &pin): Sensor(pin) {}
+
+void Altimeter::read_sensor_value() {
+  altimeter.begin();
+  altimeter.setModeAltimeter();
+  // recommended setup from sample code
+  altimeter.setOversampleRate(7); // Set Oversample to the recommended 128
+  altimeter.enableEventFlags(); // Enable all three pressure and temp even
```

Comment:
Does this stuff need to be done every time the sensor values are read?

Reply:
Good catch! Moved it to the constructor instead.